.github/scripts/common.sh

```diff
@@ -0,0 +1,206 @@
+#!/bin/bash
+
+. .github/scripts/env.sh
+
+# Colors
+RED="0;31"
+LT_BROWN="1;33"
+LT_BLUE="1;34"
+
+ecabal() {
```

Comment:
why?

Reply:
It's one of the most common shell programming patterns to wrap a binary call in a function so you can adjust error behavior and other things.